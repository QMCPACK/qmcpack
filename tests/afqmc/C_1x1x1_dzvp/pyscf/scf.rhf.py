#! /usr/bin/env python3

import numpy
from functools import reduce
from pyscf.pbc import gto, scf
from pyscf.pbc import tools as pbctools

alat0 = 3.6

cell = gto.Cell()
cell.a = (numpy.ones((3,3))-numpy.eye(3))*alat0/2.0
cell.atom = (('C',0,0,0),('C',numpy.array([0.25,0.25,0.25])*alat0))
cell.basis = 'gth-dzvp'
cell.pseudo = 'gth-pade'
cell.gs = [10]*3
cell.verbose = 5
cell.build()

nk = [1,1,1]
kpts = cell.make_kpts(nk)

mf = scf.KRHF(cell)
mf.chkfile = 'scf.dump'
ehf = mf.kernel()

import h5py

from pyscftools import integrals_from_chkfile
hcore = mf.get_hcore()                                   # obtain and store core hamiltonian
fock = (hcore + mf.get_veff())                           # store fock matrix (required with orthoAO)
X,nmo_per_kpt = integrals_from_chkfile.getOrthoAORotation(cell,kpts,1e-8)      # store rotation to orthogonal PAO basis
with h5py.File(mf.chkfile) as fh5:
  fh5['scf/hcore'] = hcore
  fh5['scf/fock'] = fock
  fh5['scf/orthoAORot'] = X
  fh5['scf/nmo_per_kpt'] = nmo_per_kpt

integrals_from_chkfile.eri_to_h5("choldump", "./scf.dump", orthoAO=False, gtol=1e-5, gtol_chol=1e-5)

