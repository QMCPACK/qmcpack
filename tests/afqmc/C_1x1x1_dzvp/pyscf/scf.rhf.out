#INFO: **** input file is /p/lscratchh/mmorale/PYSCF/debug/C/afqmc_rhf/pyscf/scf.py ****
#! /usr/bin/env python3

import numpy
from functools import reduce
from pyscf.pbc import gto, scf 
from pyscf.pbc import tools as pbctools

alat0 = 3.6

cell = gto.Cell()
cell.a = (numpy.ones((3,3))-numpy.eye(3))*alat0/2.0
cell.atom = (('C',0,0,0),('C',numpy.array([0.25,0.25,0.25])*alat0))
cell.basis = 'gth-dzvp'
cell.pseudo = 'gth-pade'
cell.gs = [10]*3  
cell.verbose = 4
cell.build()

nk = [1,1,1]
kpts = cell.make_kpts(nk)

mf = scf.KRHF(cell)
mf.chkfile = 'scf.dump'
ehf = mf.kernel()

import h5py

from pyscftools import integrals_from_chkfile
hcore = mf.get_hcore()                                   # obtain and store core hamiltonian
fock = (hcore + mf.get_veff())                           # store fock matrix (required with orthoAO)
X,nmo_per_kpt = integrals_from_chkfile.getOrthoAORotation(cell,kpts,1e-8)      # store rotation to orthogonal PAO basis
with h5py.File(mf.chkfile) as fh5:
  fh5['scf/hcore'] = hcore
  fh5['scf/fock'] = fock
  fh5['scf/orthoAORot'] = X
  fh5['scf/nmo_per_kpt'] = nmo_per_kpt

integrals_from_chkfile.eri_to_h5("choldump", "./scf.dump", orthoAO=False, gtol=1e-5, gtol_chol=1e-5)

#INFO: ******************** input file end ********************


System: ('Linux', 'quartz770', '3.10.0-693.17.1.1chaos.ch6.x86_64', '#1 SMP Fri Jan 26 13:23:01 PST 2018', 'x86_64', 'x86_64')  Threads 1
Python 2.7.14 (default, Jan 17 2018, 10:04:29) 
[GCC 4.9.3]
numpy 1.13.3  scipy 1.0.0
Date: Tue May  1 15:47:17 2018
PySCF version 1.4.6
PySCF path  /usr/gapps/afqmc/codes/pyscf_repository/pyscf_latest/pyscf
GIT ORIG_HEAD 20899530e986e85e911359e6ebc8d9c2219e26b6
GIT HEAD      ref: refs/heads/master
GIT master branch  81606c8f384ff1da98a7aa4c817021a78302110a

[INPUT] VERBOSE 4
[INPUT] num atoms = 2
[INPUT] num electrons = 8
[INPUT] charge = 0
[INPUT] spin (= nelec alpha-beta = 2S) = 0
[INPUT] symmetry False subgroup None
[INPUT]  1 C      0.000000000000   0.000000000000   0.000000000000 AA    0.000000000000   0.000000000000   0.000000000000 Bohr
[INPUT]  2 C      0.900000000000   0.900000000000   0.900000000000 AA    1.700753512109   1.700753512109   1.700753512109 Bohr
[INPUT] ---------------- BASIS SET ---------------- 
[INPUT] l, kappa, [nprim/nctr], expnt,             c_1 c_2 ...
[INPUT] C
[INPUT] 0    0    [4    /2   ]  4.3362376436      0.1490797872    0
                                1.2881838513      -0.0292640031    0
                                0.4037767149      -0.688204051    0
                                0.1187877657      -0.3964426906    1
[INPUT] 1    0    [4    /2   ]  4.3362376436      -0.0878123619    0
                                1.2881838513      -0.27755603    0
                                0.4037767149      -0.4712295093    0
                                0.1187877657      -0.4058039291    1
[INPUT] 2    0    [1    /1   ]  0.55                 1
nuclear repulsion = -12.6692033995314
number of shells = 6
number of NR pGTOs = 42
number of NR cGTOs = 26
basis = gth-dzvp
ecp = {}
CPU time:         1.47
lattice vectors  a1 [0.000000000, 3.401507024, 3.401507024]
                 a2 [3.401507024, 0.000000000, 3.401507024]
                 a3 [3.401507024, 3.401507024, 0.000000000]
dimension = 3
Cell volume = 78.7126
rcut = 20.7795245394 (nimgs = [6 6 6])
lattice sum = 1097 cells
precision = 1e-08
pseudo = gth-pade
gs (FFT-mesh) = [10, 10, 10]
    = ke_cutoff [ 127.9523481  127.9523481  127.9523481]
ew_eta = 1.55387
ew_cut = 4.38941179256 (nimgs = [2 2 2])


******** <class 'pyscf.pbc.scf.khf.KRHF'> flags ********
method = KRHF
initial guess = minao
damping factor = 0
level shift factor = 0
DIIS = <class 'pyscf.scf.diis.CDIIS'>
DIIS start cycle = 1
DIIS space = 8
SCF tol = 1e-09
SCF gradient tol = None
max. SCF cycles = 50
direct_scf = False
chkfile to save SCF result = scf.dump
max_memory 4000 MB (current use 61 MB)


******** PBC SCF flags ********
N kpts = 1
Exchange divergence treatment (exxdiv) = ewald
    madelung (= occupied orbital energy shift) = 0.673945701372
    Total energy shift due to Ewald probe charge = -1/2 * Nelec*madelung/cell.vol = -2.69578280549
DF object = <pyscf.pbc.df.fft.FFTDF object at 0x2aaaca397450>


******** <class 'pyscf.pbc.df.fft.FFTDF'> flags ********
gs = [10, 10, 10]
len(kpts) = 1

WARN: ke_cutoff/gs (127.952 / [10, 10, 10]) is not enough for FFTDF to get integral accuracy 1e-08.
Coulomb integral error is ~ 5.3e-07 Eh.
Recomended ke_cutoff/gs are 174.5 / [12 12 12].

Set gradient conv threshold to 3.16228e-05
init E= -22.9613381570762
HOMO = 0.541386814114  LUMO = 0.794246719488
cycle= 1 E= -10.3132406069019  delta_E= 12.6  |g|= 0.144  |ddm|= 6.54
HOMO = 0.236480171479  LUMO = 1.06106258658
cycle= 2 E= -10.319264221909  delta_E= -0.00602  |g|= 0.0397  |ddm|= 1.39
HOMO = 0.235265621275  LUMO = 1.06185398844
cycle= 3 E= -10.3197771785554  delta_E= -0.000513  |g|= 0.00122  |ddm|= 0.643
HOMO = 0.234966832172  LUMO = 1.0616553101
cycle= 4 E= -10.3197776711757  delta_E= -4.93e-07  |g|= 8.04e-05  |ddm|= 0.0162
HOMO = 0.234980504752  LUMO = 1.0616772317
cycle= 5 E= -10.3197776727145  delta_E= -1.54e-09  |g|= 2.01e-06  |ddm|= 0.00257
HOMO = 0.234981109326  LUMO = 1.06167708326
cycle= 6 E= -10.3197776727154  delta_E= -8.51e-13  |g|= 8.97e-07  |ddm|= 3.51e-05
HOMO = 0.234981537926  LUMO = 1.06167692829
Extra cycle  E= -10.3197776727156  delta_E= -2.04e-13  |g|= 1.29e-07  |ddm|= 2.51e-05
converged SCF energy = -10.3197776727156
Total number of orbitals:  26
Time to reach cholesky: 0.05
 Approx. total memory required:  0.18657553196  GB.
Generating orbital products.  0.22
Time to generate orbital products:  1.18
Time to generate diagonal (initial residual): 0.14
0 0.106031063572 0.01 0.0 0.0 0.01 0.0 0.0
1 0.0948268660587 0.0 0.0 0.0 0.0 0.0 0.0
2 0.0893423254697 0.01 -0.01 0.0 0.0 0.0 0.0
3 0.0738357035514 0.01 0.0 0.01 0.0 0.0 0.0
4 0.0718680180379 0.01 0.0 0.01 0.0 0.0 0.0
5 0.0637053946818 0.01 0.0 0.01 0.0 0.0 0.0
6 0.0637053946818 0.01 0.0 0.01 0.0 0.0 0.0
7 0.0619640729042 0.01 0.0 0.01 0.0 0.0 0.0
8 0.061382181672 0.01 0.0 0.01 0.0 0.0 0.0
9 0.0552712915162 0.01 0.0 0.01 0.0 0.0 0.0
10 0.0549743434431 0.01 0.0 0.01 0.0 0.0 0.0
11 0.0511332795847 0.01 0.0 0.01 0.0 0.0 0.0
12 0.0500744926524 0.01 0.0 0.01 0.0 0.0 0.0
13 0.0475714579466 0.01 0.0 0.01 0.0 0.0 0.0
14 0.0359638735639 0.01 0.0 0.01 0.0 0.0 0.0
15 0.0350490336395 0.01 0.0 0.01 0.0 0.0 0.0
16 0.0333882901166 0.01 0.0 0.01 0.0 0.0 0.0
17 0.0324638540187 0.01 0.0 0.01 0.0 0.0 0.0
18 0.0313703184264 0.01 0.0 0.01 0.0 0.0 0.0
19 0.0307259117471 0.01 0.0 0.01 0.0 0.0 0.0
20 0.0284019480206 0.01 0.0 0.01 0.0 0.0 0.0
21 0.0282783764196 0.01 0.0 0.01 0.0 0.0 0.0
22 0.0260614160746 0.01 0.0 0.01 0.0 0.0 0.0
23 0.0256260046181 0.01 0.0 0.01 0.0 0.0 0.0
24 0.0251114201588 0.01 0.0 0.01 0.0 0.0 0.0
25 0.0241363286569 0.01 0.0 0.01 0.0 0.0 0.0
26 0.0230440377272 0.01 0.0 0.0 0.01 0.0 0.0
27 0.0223838299222 0.01 0.0 0.0 0.01 0.0 0.0
28 0.0221627652699 0.0 0.0 0.0 0.0 0.0 0.0
29 0.0199509716162 0.01 -0.01 0.0 0.0 0.0 0.0
30 0.0190245608414 0.01 0.0 0.01 0.0 0.0 0.0
31 0.0178883030484 0.01 0.0 0.01 0.0 0.0 0.0
32 0.0172297888234 0.01 0.0 0.01 0.0 0.0 0.0
33 0.0166897332107 0.01 0.0 0.01 0.0 0.0 0.0
34 0.015739766529 0.01 0.0 0.01 0.0 0.0 0.0
35 0.0146231087579 0.01 0.0 0.01 0.0 0.0 0.0
36 0.0143329797388 0.01 0.0 0.01 0.0 0.0 0.0
37 0.0124250610187 0.01 0.0 0.01 0.0 0.0 0.0
38 0.0120426853973 0.01 0.0 0.01 0.0 0.0 0.0
39 0.0120339575423 0.01 0.0 0.01 0.0 0.0 0.0
40 0.0119917931539 0.01 0.0 0.01 0.0 0.0 0.0
41 0.0115650318185 0.01 0.0 0.01 0.0 0.0 0.0
42 0.0115205133982 0.01 0.0 0.01 0.0 0.0 0.0
43 0.0113262972395 0.01 0.0 0.01 0.0 0.0 0.0
44 0.0113083700842 0.01 0.0 0.01 0.0 0.0 0.0
45 0.0109903585376 0.01 0.0 0.01 0.0 0.0 0.0
46 0.0108465620328 0.01 0.0 0.01 0.0 0.0 0.0
47 0.010761787947 0.01 0.0 0.01 0.0 0.0 0.0
48 0.010735463138 0.01 0.0 0.01 0.0 0.0 0.0
49 0.0104405156782 0.01 0.0 0.01 0.0 0.0 0.0
50 0.0102950565043 0.01 0.0 0.01 0.0 0.0 0.0
51 0.00990460405144 0.01 0.0 0.01 0.0 0.0 0.0
52 0.00980806671309 0.01 0.0 0.01 0.0 0.0 0.0
53 0.00970296890413 0.01 0.0 0.01 0.0 0.0 0.0
54 0.00943331989198 0.01 0.0 0.01 0.0 0.0 0.0
55 0.00938771128304 0.01 0.0 0.01 0.0 0.0 0.0
56 0.00938443957404 0.01 0.0 0.01 0.0 0.0 0.0
57 0.00892330924888 0.01 0.0 0.01 0.0 0.0 0.0
58 0.00856899192134 0.01 0.0 0.01 0.0 0.0 0.0
59 0.00833500810938 0.01 0.0 0.01 0.0 0.0 0.0
60 0.00830432900193 0.01 0.0 0.01 0.0 0.0 0.0
61 0.00826884580548 0.01 0.0 0.01 0.0 0.0 0.0
62 0.00824616124029 0.01 0.0 0.01 0.0 0.0 0.0
63 0.00817800885842 0.01 0.0 0.01 0.0 0.0 0.0
64 0.0081384117203 0.01 0.0 0.01 0.0 0.0 0.0
65 0.00796234390176 0.01 0.0 0.0 0.01 0.0 0.0
66 0.00787245606949 0.01 0.0 0.0 0.01 0.0 0.0
67 0.00771499742491 0.01 0.0 0.0 0.01 0.0 0.0
68 0.00750400262967 0.0 0.0 0.0 0.0 0.0 0.0
69 0.00741243335884 0.01 -0.01 0.0 0.0 0.0 0.0
70 0.0074063231112 0.01 0.0 0.01 0.0 0.0 0.0
71 0.00733553800626 0.01 0.0 0.01 0.0 0.0 0.0
72 0.00697994348382 0.01 0.0 0.01 0.0 0.0 0.0
73 0.00664393542103 0.01 0.0 0.01 0.0 0.0 0.0
74 0.00628853831276 0.01 0.0 0.01 0.0 0.0 0.0
75 0.00623906950232 0.01 0.0 0.01 0.0 0.0 0.0
76 0.00617889072282 0.01 0.0 0.01 0.0 0.0 0.0
77 0.00611007152357 0.01 0.0 0.01 0.0 0.0 0.0
78 0.00602694236387 0.01 0.0 0.01 0.0 0.0 0.0
79 0.00598347758468 0.01 0.0 0.01 0.0 0.0 0.0
80 0.00594025325525 0.01 0.0 0.01 0.0 0.0 0.0
81 0.00587514567254 0.01 0.0 0.01 0.0 0.0 0.0
82 0.00553873158338 0.01 0.0 0.01 0.0 0.0 0.0
83 0.00536534174754 0.01 0.0 0.01 0.0 0.0 0.0
84 0.00532298267568 0.01 0.0 0.01 0.0 0.0 0.0
85 0.00510263758496 0.01 0.0 0.01 0.0 0.0 0.0
86 0.0050940370149 0.01 0.0 0.01 0.0 0.0 0.0
87 0.00498163716416 0.01 0.0 0.01 0.0 0.0 0.0
88 0.00494910702892 0.01 0.0 0.01 0.0 0.0 0.0
89 0.00492162527351 0.01 0.0 0.01 0.0 0.0 0.0
90 0.00485357048237 0.01 0.0 0.01 0.0 0.0 0.0
91 0.0046852726388 0.01 0.0 0.01 0.0 0.0 0.0
92 0.00466803971795 0.01 0.0 0.01 0.0 0.0 0.0
93 0.00460127417815 0.01 0.0 0.01 0.0 0.0 0.0
94 0.00457843305626 0.01 0.0 0.01 0.0 0.0 0.0
95 0.00453876039151 0.01 0.0 0.01 0.0 0.0 0.0
96 0.00448515785111 0.01 0.0 0.01 0.0 0.0 0.0
97 0.0043008987299 0.01 0.0 0.01 0.0 0.0 0.0
98 0.00428809410954 0.01 0.0 0.01 0.0 0.0 0.0
99 0.00422938029332 0.01 0.0 0.01 0.0 0.0 0.0
100 0.00419340100846 0.01 0.0 0.0 0.01 0.0 0.0
101 0.00412926698945 0.01 0.0 0.0 0.01 0.0 0.0
102 0.00410210992564 0.01 0.0 0.0 0.0 0.01 0.0
103 0.00407214924233 0.0 0.0 0.0 0.0 0.0 0.0
104 0.00386000383315 0.01 0.0 0.01 0.0 0.0 0.0
105 0.00383071626302 0.01 0.0 0.01 0.0 0.0 0.0
106 0.00381351502877 0.01 0.0 0.01 0.0 0.0 0.0
107 0.00367933745093 0.01 0.0 0.01 0.0 0.0 0.0
108 0.00363503451787 0.01 0.0 0.01 0.0 0.0 0.0
109 0.00352876962324 0.01 0.0 0.01 0.0 0.0 0.0
110 0.00351820699847 0.01 0.0 0.01 0.0 0.0 0.0
111 0.00344691767025 0.01 0.0 0.01 0.0 0.0 0.0
112 0.00336333029064 0.01 0.0 0.01 0.0 0.0 0.0
113 0.00333826784855 0.01 0.0 0.01 0.0 0.0 0.0
114 0.00327359092623 0.01 0.0 0.01 0.0 0.0 0.0
115 0.00323271149091 0.01 0.0 0.01 0.0 0.0 0.0
116 0.00307141592683 0.01 0.0 0.01 0.0 0.0 0.0
117 0.0030167943297 0.01 0.0 0.01 0.0 0.0 0.0
118 0.00278323073268 0.01 0.0 0.01 0.0 0.0 0.0
119 0.00267410571394 0.01 0.0 0.01 0.0 0.0 0.0
120 0.0026045690738 0.01 0.0 0.01 0.0 0.0 0.0
121 0.00259180172237 0.01 0.0 0.01 0.0 0.0 0.0
122 0.00253682045437 0.01 0.0 0.01 0.0 0.0 0.0
123 0.00252627471501 0.01 0.0 0.01 0.0 0.0 0.0
124 0.00252168670903 0.01 0.0 0.01 0.0 0.0 0.0
125 0.00243420620984 0.01 0.0 0.0 0.01 0.0 0.0
126 0.00240889335498 0.01 0.0 0.0 0.01 0.0 0.0
127 0.00237943103326 0.0 0.0 0.0 0.0 0.0 0.0
128 0.00237345721456 0.01 -0.01 0.0 0.0 0.0 0.0
129 0.00236813766969 0.01 0.0 0.01 0.0 0.0 0.0
130 0.00234450855361 0.01 0.0 0.01 0.0 0.0 0.0
131 0.0023362897355 0.01 0.0 0.01 0.0 0.0 0.0
132 0.0022871200299 0.01 0.0 0.01 0.0 0.0 0.0
133 0.00224838652123 0.01 0.0 0.01 0.0 0.0 0.0
134 0.00222991649846 0.01 0.0 0.01 0.0 0.0 0.0
135 0.00219450216665 0.01 0.0 0.01 0.0 0.0 0.0
136 0.00203157181779 0.01 0.0 0.01 0.0 0.0 0.0
137 0.00203066103113 0.01 0.0 0.01 0.0 0.0 0.0
138 0.00198249613744 0.01 0.0 0.01 0.0 0.0 0.0
139 0.00194463303638 0.01 0.0 0.01 0.0 0.0 0.0
140 0.00188891265089 0.01 0.0 0.01 0.0 0.0 0.0
141 0.00182037003446 0.01 0.0 0.01 0.0 0.0 0.0
142 0.00181616229915 0.01 0.0 0.01 0.0 0.0 0.0
143 0.00176058990822 0.01 0.0 0.01 0.0 0.0 0.0
144 0.00162509387615 0.01 0.0 0.01 0.0 0.0 0.0
145 0.0015983592809 0.01 0.0 0.01 0.0 0.0 0.0
146 0.00157263236096 0.01 0.0 0.01 0.0 0.0 0.0
147 0.00148981080022 0.01 0.0 0.01 0.0 0.0 0.0
148 0.00145344039536 0.01 0.0 0.01 0.0 0.0 0.0
149 0.00139420999712 0.01 0.0 0.01 0.0 0.0 0.0
150 0.00130302228237 0.01 0.0 0.01 0.0 0.0 0.0
151 0.00127946837631 0.01 0.0 0.01 0.0 0.0 0.0
152 0.00120812377798 0.01 0.0 0.0 0.01 0.0 0.0
153 0.00120379733235 0.01 0.0 0.0 0.01 0.0 0.0
154 0.00117556994298 0.01 0.0 0.0 0.01 0.0 0.0
155 0.00113309652427 0.0 0.0 0.0 0.0 0.0 0.0
156 0.00107512883873 0.01 -0.01 0.0 0.0 0.0 0.0
157 0.00107012358897 0.01 0.0 0.01 0.0 0.0 0.0
158 0.00103001665718 0.01 0.0 0.01 0.0 0.0 0.0
159 0.00102256364313 0.01 0.0 0.01 0.0 0.0 0.0
160 0.000974475611791 0.01 0.0 0.01 0.0 0.0 0.0
161 0.000926697277484 0.01 0.0 0.01 0.0 0.0 0.0
162 0.000888965556516 0.01 0.0 0.01 0.0 0.0 0.0
163 0.000857377610396 0.01 0.0 0.01 0.0 0.0 0.0
164 0.000838257640518 0.01 0.0 0.01 0.0 0.0 0.0
165 0.000801247718146 0.01 0.0 0.01 0.0 0.0 0.0
166 0.000775953394985 0.01 0.0 0.01 0.0 0.0 0.0
167 0.000725459743124 0.01 0.0 0.01 0.0 0.0 0.0
168 0.00068865274667 0.01 0.0 0.01 0.0 0.0 0.0
169 0.000662405229252 0.01 0.0 0.01 0.0 0.0 0.0
170 0.000650064032185 0.01 0.0 0.01 0.0 0.0 0.0
171 0.000609381036187 0.01 0.0 0.01 0.0 0.0 0.0
172 0.000601608404146 0.01 0.0 0.01 0.0 0.0 0.0
173 0.000566991201813 0.01 0.0 0.01 0.0 0.0 0.0
174 0.000528715904605 0.01 0.0 0.01 0.0 0.0 0.0
175 0.000526276094324 0.01 0.0 0.01 0.0 0.0 0.0
176 0.00052552759695 0.01 0.0 0.01 0.0 0.0 0.0
177 0.000522588516768 0.01 0.0 0.0 0.01 0.0 0.0
178 0.000517107720274 0.01 0.0 0.0 0.01 0.0 0.0
179 0.000486534890618 0.0 0.0 0.0 0.0 0.0 0.0
180 0.000484894128799 0.01 -0.01 0.0 0.0 0.0 0.0
181 0.000458048425358 0.01 0.0 0.01 0.0 0.0 0.0
182 0.000414946154585 0.01 0.0 0.01 0.0 0.0 0.0
183 0.000409110552775 0.01 0.0 0.01 0.0 0.0 0.0
184 0.000400304483745 0.01 0.0 0.01 0.0 0.0 0.0
185 0.000389802521133 0.01 0.0 0.01 0.0 0.0 0.0
186 0.000385553590616 0.01 0.0 0.01 0.0 0.0 0.0
187 0.000326313002719 0.01 0.0 0.01 0.0 0.0 0.0
188 0.000325983485783 0.01 0.0 0.01 0.0 0.0 0.0
189 0.00031768430542 0.01 0.0 0.01 0.0 0.0 0.0
190 0.000307446916179 0.01 0.0 0.01 0.0 0.0 0.0
191 0.000298360240113 0.01 0.0 0.01 0.0 0.0 0.0
192 0.000293785836338 0.01 0.0 0.01 0.0 0.0 0.0
193 0.000290388376881 0.01 0.0 0.01 0.0 0.0 0.0
194 0.000287095461812 0.01 0.0 0.01 0.0 0.0 0.0
195 0.000277206567885 0.01 0.0 0.01 0.0 0.0 0.0
196 0.000272903328568 0.01 0.0 0.0 0.01 0.0 0.0
197 0.000272583807084 0.01 0.0 0.0 0.01 0.0 0.0
198 0.000261897155219 0.0 0.0 0.0 0.0 0.0 0.01
199 0.00022947697225 0.0 0.0 0.0 0.0 0.0 0.0
200 0.000217241386017 0.01 -0.01 0.0 0.0 0.0 0.0
201 0.000187423552292 0.01 -0.01 0.0 0.0 0.0 0.0
202 0.000166034710113 0.01 0.0 0.01 0.0 0.0 0.0
203 0.00015502602045 0.01 0.0 0.01 0.0 0.0 0.0
204 0.000153604146425 0.01 0.0 0.01 0.0 0.0 0.0
205 0.000153541542984 0.01 0.0 0.01 0.0 0.0 0.0
206 0.000135357980453 0.01 0.0 0.01 0.0 0.0 0.0
207 0.000124360600256 0.01 0.0 0.01 0.0 0.0 0.0
208 0.000118891010713 0.01 0.0 0.01 0.0 0.0 0.0
209 0.000110687136349 0.01 0.0 0.01 0.0 0.0 0.0
210 8.02988966969e-05 0.01 0.0 0.01 0.0 0.0 0.0
211 7.6617145015e-05 0.01 0.0 0.01 0.0 0.0 0.0
212 6.86947195252e-05 0.01 0.0 0.01 0.0 0.0 0.0
213 6.35038099945e-05 0.01 0.0 0.01 0.0 0.0 0.0
214 5.97562352705e-05 0.01 0.0 0.01 0.0 0.0 0.0
215 5.82747998668e-05 0.01 0.0 0.01 0.0 0.0 0.0
216 5.02433363116e-05 0.01 0.0 0.01 0.0 0.0 0.0
217 4.48325499988e-05 0.01 0.0 0.01 0.0 0.0 0.0
218 4.47222918367e-05 0.01 0.0 0.01 0.0 0.0 0.0
219 3.71788831536e-05 0.01 0.0 0.01 0.0 0.0 0.0
220 3.47204240072e-05 0.01 0.0 0.01 0.0 0.0 0.0
221 3.46198171534e-05 0.01 0.0 0.01 0.0 0.0 0.0
222 3.19272553508e-05 0.01 0.0 0.01 0.0 0.0 0.0
223 2.83474123574e-05 0.01 0.0 0.01 0.0 0.0 0.0
224 1.637015419e-05 0.01 0.0 0.01 0.0 0.0 0.0
225 1.5268185613e-05 0.01 0.0 0.01 0.0 0.0 0.0
226 1.08932072095e-05 0.01 0.0 0.01 0.0 0.0 0.0
227 1.0124262418e-05 0.01 0.0 0.01 0.0 0.0 0.0
228 9.86749415793e-06 0.01 0.0 0.01 0.0 0.0 0.0
Adding ewald correction to the energy:  -2.69578280549
